mcxisp/mcxisp.c

```diff
@@ -0,0 +1,608 @@
+/*
+ * Phoenix-RTOS
+ *
+ * mcxisp - tool for MCX N94x series ISP
+ *
+ * Copyright 2024 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <stddef.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <termios.h>
+#include <sys/stat.h>
+
+/* clang-format off*/
```

Comment:
Missing space, I guess that's why c-f still complains
```suggestion
/* clang-format off */
```